source/preprocessor.tex

```diff
@@ -1430,7 +1429,7 @@
 \terminal{_Pragma} \terminal{(} string-literal \terminal{)}
 \end{ncbnf}
 
-is processed as follows: The string literal is \term{destringized}
+is processed as follows: The string literal is \defn{destringized}
```

Comment:
We should index this as "destringization" or similar.

Reply:
Sure.